src/components/LoadingIndicator/LoadingIndicator.tsx

```diff
@@ -0,0 +1,81 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import { Oval } from 'react-loader-spinner';
+import styles from './LoadingIndicator.module.css';
+
+export type Props = {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+
+  /**
+   * Layout size of the loader. This affects the overall size and associated
+   * stroke width.
+   */
+  size?: 'sm' | 'md' | 'lg';
+
+  /**
+   * Aria label of the oval. Default is oval-loading. Will be overridden if ariaLabel is passed in props
+   * (Shadowed from OvalProps)
+   *
+   * @see https://mhnpd.github.io/react-loader-spinner/docs/components/oval
```

Comment:
Do we need the `@see` jsdoc tag here? It'd be nice to have this url show up on the storybook docs page.

Reply:
Interesting, i didn't realize the `@see` was causing it to be hidden by storybook. If i remove that jsdoc directive, it will show up in storybook? I can test that.

Reply:
<img width="537" alt="Screen Shot 2022-12-16 at 11 04 13 " src="https://user-images.githubusercontent.com/3056447/208150900-46e1cfac-f038-42e1-9753-111d8f469ffc.png">

Reply:
Yeah for some reason storybook just cannot handle those `@` tags and won't render it or anything after it 🤷